### PR TITLE
Modifying output of gpcheckcat to include the name of the test on the…

### PR DESCRIPTION
**Pull request by royc1**
… same line to make grepping easier

### CODE REVIEW

File: gpMgmt/bin/lib/gpcheckcat [hunk 1]
```diff
@@ -3607,7 +3607,7 @@ def runTestCatname(cat):
         GV.elapsedTime += elapsed
         elapsed = str(datetime.timedelta(seconds=elapsed))[:-4]
         GV.totalTestRun += 1
-        myprint('Total runtime for this test: %s' % (elapsed))
+        myprint("Total runtime for test '%s': %s" % (t, elapsed))
```
Comment (kaknikhil) [inline gpMgmt/bin/lib/gpcheckcat]: does it make sense to add "cat.getTableName()" to make the output even more verbose ?

Comment (royc1) [inline gpMgmt/bin/lib/gpcheckcat]: That would be repetitive as test name is already there. Another change has that. Here it is running a for loop for all the test names and getting each test name as t which does not look good. I did not  change the original code, but I think I will modify the variable name t to some more readable name.

**Comment by danielgustafsson** [comment]
Looks good to me.

**Comment by ascherbaum-pivotal** [comment]
I like it, looks good.

**Pull request merged**